### PR TITLE
feat: even more dynamic tqdm pbar for update_media!

### PR DESCRIPTION
**Pull request by AndreasPB**
# Description

Lets us keep track of how the Internal job is doing from the CLI

billede

#503

### CODE REVIEW

File: internal/routes.go [hunk 1]
```diff
@@ -130,7 +130,11 @@ func processMedia(c *gin.Context) {
 		c.JSON(http.StatusInternalServerError, gin.H{"info": fmt.Sprint(task.TaskUID) + ": " + err.Error()})
 	}
 
-	c.JSON(http.StatusOK, gin.H{"info": fmt.Sprintf("Fetched and inserted %d media into meilisearch and skipped %d", len(medias), failedMedia)})
+	c.JSON(http.StatusOK, gin.H{
+		"info":       fmt.Sprintf("Fetched and inserted %d media into meilisearch and skipped %d", len(medias), failedMedia),
+		"successful": len(medias),
+		"skipped":    failedMedia,
```
Comment (nullxDEADBEEF) [inline internal/routes.go:136]: Don't they need to be printed somewhere in Python?

**Review by nullxDEADBEEF — CHANGES_REQUESTED** [review]
See comment

File: internal/routes.go (hunk 1 above)
Comment (AndreasPB) [reply internal/routes.go:136]: They are - check the pbar.set_postfix()

Comment (nullxDEADBEEF) [reply internal/routes.go:136]: Yeah they are printed in the progress bar, but not from the json sent from Go -> Python

Comment (nullxDEADBEEF) [reply internal/routes.go:136]: not sure if we need them, if we can see it the numbers in the `info` field in the json?

Comment (nullxDEADBEEF) [reply internal/routes.go:136]: wait i'm stupid as hell, my bad xD

**Review by nullxDEADBEEF — APPROVED** [review]
LGTM :shipit:

**Pull request merged**